packages/components/src/templates/next/components/internal/Navbar/Navbar.tsx

```diff
@@ -248,91 +194,36 @@ export const Navbar = ({
           {search.type === "searchSG" && (
             <SearchSGInputBox
               clientId={search.clientId}
+              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
               ScriptComponent={ScriptComponent}
             />
           )}
         </div>
       )}
-
-      {/* Navigation items, first level (for mobile/tablet) */}
-      {isHamburgerOpen && openNavItemIdx === -1 && (
-        <div
-          className="block lg:hidden"
-          style={{
-            height: `calc(100vh - ${siteHeaderBottomY}px)`,
-          }}
-        >
-          <ul className="px-6 pt-4">
-            {items.map(({ name, url, items }, idx) => {
-              if (!items || items.length === 0) {
-                return (
-                  <li key={Math.random()} className="w-full py-3">
-                    <LinkComponent
-                      className="text-md block w-full text-content hover:text-content-medium"
-                      href={url}
-                    >
-                      {name}
-                    </LinkComponent>
-                  </li>
-                )
-              }
-
-              return (
-                <li key={Math.random()} className="w-full py-3">
-                  <button
-                    onClick={() => setOpenNavItemIdx(idx)}
-                    className="w-full"
-                  >
-                    <div className="flex w-full flex-row justify-between">
-                      <p className="text-md text-content">{name}</p>
-                      <BiChevronRight className="text-2xl" />
-                    </div>
-                  </button>
-                </li>
-              )
-            })}
-          </ul>
-        </div>
-      )}
-
-      {/* Navigation items, second level (for mobile/tablet) */}
-      {isHamburgerOpen && openNavItemIdx !== -1 && (
+      {isHamburgerOpen && (
         <div
-          className="block lg:hidden"
+          ref={mobileMenuRef}
+          className="fixed inset-0"
```

Comment:
nit: let you decide if you want to hide the mobile menu on resize to desktop

Reply:
not going to do it, not worth it to add extra javascript to do this

Reply:
Hmm was actually thinking of something like `fixed lg:hidden` but yeah I think let's just leave it bah can tackle next time